vocode/streaming/models/synthesizer.py

```diff
@@ -226,3 +227,17 @@ class PollySynthesizerConfig(SynthesizerConfig, type=SynthesizerType.POLLY.value
     language_code: str = DEFAULT_POLLY_LANGUAGE_CODE
     voice_id: str = DEFAULT_POLLY_VOICE_ID
     sampling_rate: int = DEFAULT_POLLY_SAMPLING_RATE
+
+
+DEFAULT_CARTESIA_MODEL_ID = 'upbeat-moon'
+DEFAULT_CARTESIA_VOICE_ID = '5345cf08-6f37-424d-a5d9-8ae1101b9377'
+DEFAULT_CARTESIA_AUDIO_ENCODING = AudioEncoding.LINEAR16
```

Comment:
```suggestion
```

Reply:
shouldn't need to specify default audio encoding / sampling rate (the output device will take care of this!)